site/docs/design_tokens/border-radius.mdx

```diff
@@ -0,0 +1,58 @@
+---
+name: Border Radius
+menu: Design tokens
+route: /design-tokens/border-radius
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+
+# Border Radius
+
+<LargeParagraph>The border-radius tokens are used to give elements rounded corners.</LargeParagraph>
```

Comment:
> The border-radius tokens are used to give elements rounded corners.

→ The border-radius tokens are used control roundness of element's corners.

Kulmathan ei välttämättä ole pyöreitä, vaan useinmiten teräviä.